Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Tabular/Merchant/MerchantServiceUsageConfig.hs

```diff
@@ -36,6 +37,7 @@ mkPersist
   [defaultQQ|
     MerchantServiceUsageConfigT sql=merchant_service_usage_config
       merchantId MerchantTId
+      initiateCall CallService
```

Comment:
Add migration. Also add migration to add Exotel configs in MerchantServiceConfig